src/app/core/learning-object-module/search/search.service.ts

```diff
@@ -1,9 +1,100 @@
 import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { LearningObject } from '@entity';
+import { catchError } from 'rxjs/operators';
+import { Subject, throwError } from 'rxjs';
+import { Query } from 'app/interfaces/query';
+import * as querystring from 'querystring';
+import { SEARCH_ROUTES } from 'app/core/learning-object-module/search/search.routes';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class SearchService {
+  needsChange$: Subject<void> = new Subject();
 
```

Comment:
This route needs to be moved into the search module also. This comes from the learning object search module route handler in clark-service. 

https://github.com/Cyber4All/clark-service/blob/main/src/modules/learning-object-module/search/search.routehandler.ts

![Screenshot 2024-09-16 at 10 09 34 AM](https://github.com/user-attachments/assets/148ac766-ad1c-4894-a132-b5c22bcc10de)